### DISCUSSION TITLE
How to load unpacked

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#740] [Help] [answered]
Could you post a video on the steps to load the source code into chrome from windows?

**Comment by TheColaber** [comment, marked as answer]
You mean into a file editor or just load it into chrome://extensions ?

**Reply by ghost** [reply]
> You mean into a file editor or just load it into chrome://extensions ?

chrome://extensions

**Reply by TheColaber** [reply]
K one sec.

**Reply by TheColaber** [reply]
On https://github.com/ScratchAddons/ScratchAddons click "code" and then "download zip" - It should go to your downloads folder.
![image](https://user-images.githubusercontent.com/72760579/98978202-9d4c4a00-24e7-11eb-8e52-8df1c25d3113.png)
Then go to chrome://extensions
Make sure developer mode is on: (It is in the RIGHT corner)
![image](https://user-images.githubusercontent.com/72760579/98978015-4e9eb000-24e7-11eb-8a8c-802110e01b6b.png)
Click on "Load Unpacked".
![image](https://user-images.githubusercontent.com/72760579/98978099-6b3ae800-24e7-11eb-94ae-cb848a538ab5.png)
Select the ScratchAddons file.

Edit: If you see an "errors" button, don't mind that, it is just giving an error because we have some code that is for firefox.

**Reply by retronbv** [reply]
i think they mean a video

> Could you post a video on the steps to load the source code into chrome from windows?

> Make sure developer mode is on: (It is in the left corner) 

its the right corner not left

**Reply by ghost** [reply]
I don't see the folder anywhere.

**Reply by ghost** [reply]
> i think they mean a video
> 
> > Could you post a video on the steps to load the source code into chrome from windows?
> 
> edit: its the right corner not left
No it's in the left corner

**Reply by retronbv** [reply]
what folder

**Reply by TheColaber** [reply]
ignore this.

**Reply by ghost** [reply]
> what folder

The scratch addons-master zip

**Reply by TheColaber** [reply]
> > what folder
> 
> The scratch addons-master zip

Where did you save it? You could have saved it somewhere else...

**Reply by ghost** [reply]
It saved into ThisPC/Videos.

**Reply by ghost** [reply]
I can't save it in a place of my choice. Windows did it for me

**Reply by TheColaber** [reply]
> It saved into ThisPC/Videos.

Then move it somewhere you can find and then try that.

**Reply by TheColaber** [reply]
> I can't save it in a place of my choice. Windows did it for me

move the file... drag it.

**Reply by ghost** [reply]
I can find the folder, but when I click Load unpacked it seems to disappear.

**Reply by TheColaber** [reply]
Please send a screenshot or recording.

**Reply by ghost** [reply]
> > I can't save it in a place of my choice. Windows did it for me
> 
> move the file... drag it.

THAT WON'T WORK

**Reply by ghost** [reply]
> > > I can't save it in a place of my choice. Windows did it for me
> > 
> > 
> > move the file... drag it.
> 
> THAT WON'T WORK

*Didn't*

**Reply by retronbv** [reply]
(doesnt work)

**Reply by ghost** [reply]
> https://drive.google.com/file/d/1y6tUtRkxg0q-E2Q3D8rsmIzPhxsybRjt/view

File don't exist bud 
nvm but it won't load

**Reply by retronbv** [reply]
aww shoot 1 sec

**Reply by ghost** [reply]
post a gif maybe

**Reply by retronbv** [reply]
https://imgur.com/a/eYQBlj2

**Reply by ghost** [reply]
A windows one please

**Reply by retronbv** [reply]
i dont have a windows pc i can access rn

**Reply by ghost** [reply]
> i dont have a windows pc i can access rn

I have chromebook but gif too small

**Reply by TheColaber** [reply]
I will do one for you later.

**Reply by ghost** [reply]
And i'm managed by parent :( so no dev mode

**Reply by ghost** [reply]
> I will do one for you later.

Nevermind I just did the same thing in the gif but in windows and it worked

**Reply by retronbv** [reply]
but mine was the answer :C

**Reply by TheColaber** [reply]
they marked this entire stack as the answer. because I started the stack, it shows me.

**Reply by retronbv** [reply]
i was joking
